Style-guide/README.md

```diff
@@ -71,8 +68,8 @@ Descriptive and consistent naming makes software easier to read and understand.
 
 - striving for clarity at the call site
 - prioritizing clarity over brevity
-- using camel case (not snake case)
-- using uppercase for types (and protocols), lowercase for everything else
+- using `camelCase` (not `snake_case`)
+- using uppercase initials for types (and protocols), lowercase first-initial for everything else
```

Comment:
Do we need examples here or is it already clear enough?

Reply:
i think it's fine given it's a swift convention